### DISCUSSION TITLE
eslint-config-next to have eslint ^9.0.0 as`peerDependencies`

### DISCUSSION THREAD

**Discussion by VincentLu91** [vercel/next.js#64269] [Ideas]
### Goals

1. update eslint-config-next to support the new version of eslint
2. enable Next.js applications to be updated and secure
3. reduce dependency issues

### Non-Goals

N/A

### Background

https://github.com/eslint/eslint/issues/18297

### Proposal

Please see `background` field

**Comment by alvinsjoy** [comment]
@VincentLu91 I guess this discussion is not getting any attention. It might be better to create an issue.

**Reply by VincentLu91** [reply]
@alvinsjoy 
this is raised. it really needs attention. I was the only one who seemed to have the experience but now i'm starting to see others experiencing the same, in Discord

Issue:
https://github.com/vercel/next.js/issues/64833#event-12548915205

**Reply by alvinsjoy** [reply]
@VincentLu91 it seems that the issue was closed automatically. I have reopened the issue at #64853

**Comment by leerob** [comment]
ESLint 9 is supported as of the Next.js 15 RC 2. v15 stable release coming soon 🙏 

https://x.com/nextjs/status/1846276572812124565

**Reply by VincentLu91** [reply]
thanks for letting us know @leerob !

**Reply by alvinsjoy** [reply]
Thanks for the update
